gui/src/components/Taipy/Chart.tsx

```diff
@@ -62,6 +64,9 @@ interface ChartProp extends TaipyActiveProps, TaipyChangeProps {
     render?: boolean;
     defaultRender?: boolean;
     decimator?: string;
+    template?: string;
```

Comment:
same based branch as the chart theme pr I guess

Reply:
Indeed, yes, I'll merge it after the theme one